cinema/views.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):

```

Comment:
Consider adding docstrings to the `movie_list` function to describe its purpose and the HTTP methods it supports. This will improve code readability and maintainability.